addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -149,6 +149,10 @@ export(NodePath) var animation_player_node: NodePath = "" \
 # as a camera target
 export(NodePath) var camera_node
 
+# Custom data dictionary to ease customization and custom command creation.
+# Avoid name collision using proper key names.
+export(Dictionary) var custom_data = {}
```

Comment:
I think you included code from PR#705, which was merged a bit earlier. You may end up with conflicts here. `Git rebase` is probably needed here ;)

Reply:
This codeblock is in the current develop branch - https://github.com/godot-escoria/escoria-demo-game/blob/develop/addons/escoria-core/game/core-scripts/esc_item.gd#L154 ?